ts/packages/ts/src/commands/commands.test.ts

```diff
@@ -222,4 +224,179 @@ describe(`commands`, () => {
       }
     });
   });
+  describe(`ExportCommand`, () => {
+    describe(`Keys Mutation Callback`, () => {
+      test('Export without keys mutation callback', async () => {
+        const evalResult = await new EvalCommand({
+          store: store1,
+          set: set1,
+          schema: new ConfigSchema('mutate', {
+            KEY0: {
+              type: 'String',
+            },
+            KEY1: {
+              type: 'String',
+            },
+          }),
+          configs: {
+            KEY0: 'KEY0',
+            KEY1: 'KEY1',
+          },
+        }).run();
+        const exportedConfigs = await new ExportCommand({ pipe: evalResult }).run();
+        expect(exportedConfigs).toStrictEqual({ KEY0: 'KEY0', KEY1: 'KEY1' });
+      });
+    });
+    test('Export with keys mutation callback', async () => {
+      const evalResult = await new EvalCommand({
+        store: store1,
+        set: set1,
+        schema: new ConfigSchema('mutate', {
+          KEY0: {
+            type: 'String',
+          },
+          KEY1: {
+            type: 'String',
+          },
+        }),
+        configs: {
+          KEY0: 'KEY0',
+          KEY1: 'KEY1',
+        },
+      }).run();
+      const exportedConfigs = await new ExportCommand({ pipe: evalResult, keys: (key) => `MY_${key}` }).run();
+      expect(exportedConfigs).toStrictEqual({ MY_KEY0: 'KEY0', MY_KEY1: 'KEY1' });
+    });
+    test('Export with bad keys mutation callback - returns non-string', async () => {
+      expect.assertions(1);
+      const evalResult = await new EvalCommand({
+        store: store1,
+        set: set1,
+        schema: new ConfigSchema('mutate', {
+          KEY0: {
+            type: 'String',
+          },
+          KEY1: {
+            type: 'String',
+          },
+        }),
+        configs: {
+          KEY0: 'KEY0',
+          KEY1: 'KEY1',
+        },
+      }).run();
+      await expect(
+        new ExportCommand({
+          pipe: evalResult,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-expect-error
```

Comment:
NIT: Cleans up the disable next line. Check if it works for you.
```suggestion
          // @ts-expect-error - should throw ConfigError
```